### DISCUSSION TITLE
Context Menu not working with Taskbar on Top

### DISCUSSION THREAD

**Discussion by rafaelmendoncavaz** [valinet/ExplorerPatcher#3633] [Q&A] [answered]
Just started using this patcher, but it seems that the right click menu on taskbar icons works on any position excepts at the top. I tested on the sides and default, but when I use it at the top it doesnt work. Is it just me or its a common case for that position?

Also... is there a way to fix it?

![image](https://github.com/user-attachments/assets/b2e8b859-940a-423a-82cd-d29677cb11d9)

**Comment by milestprower92** [comment, marked as answer]
Install the latest pre release:
https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3880.66.5_5094108

**Reply by rafaelmendoncavaz** [reply]
Worked! Ty
